src/Context/FixtureContext.php

```diff
@@ -687,6 +695,42 @@ class_exists($extension) && is_subclass_of($extension, Extension::class),
         }
     }
 
+    /**
+     * Get the module that represents the application.
+     */
+    protected function getAppModule(): Module
+    {
+        $project = ModuleManifest::config()->get('project') ?: 'mysite';
+        $mysite = ModuleLoader::getModule($project);
+        Assert::assertNotNull($mysite, 'Project exists');
+        return $mysite;
+    }
+
+    /**
+     * Get the main folder for the application.
+     */
+    protected function getDestinationTemplatePath(string $filename, bool $createMissingDirs): string
+    {
+        $app = $this->getAppModule();
+        $basePath = $app->getPath();
+        $destPath = $app->getResource("templates/{$filename}")->getPath();
```

Comment:
There's an argument to be made that this should be creating a new temporary test theme instead - which I'd be open to doing if that's preferred.